Core/nextspace-core.spec

```diff
@@ -64,6 +65,8 @@ unset LDFLAGS
 
 # Build gnustep-make to include in -devel package
 cd tools-make-make-%{MAKE_VERSION}
+sed -i 's|^AC_PATH_PROG|PATH=/usr/bin:/bin AC_PATH_PROG|' configure.ac
+autoconf
```

Comment:
What for? `configure` already exists in gnustep-make sources. No need to regenerate it.

Reply:
The problem with the `AC_PATH_PROG` macro is that it behaves differently if you are building as root. In this particular case this leads to a package dependency to `/usr/sbin/bash` because `/usr/sbin` is searched first. 

I build as root in a docker container because this allows me to install packages without having to elevate privileges. `sudo` is always a bit tricky in a script without a terminal.

Reply:
`/usr/sbin/bash` exists on my Fedora 42. Maybe it's better to fix your docker container?

Reply:
The docker image I am using is the official fedora image. 
And, yes, `/usr/sbin/bash` exists. That's not the issue.
An RPM dependency to `/usr/sbin/bash` is generated. And `dnf` or `yum` cannot install `bash` by this dependency.
I.e. I get an RPM that is uninstallable because of this dependency. If you `rpm -qR` the RPM from your own build it will not list `/usr/sbin/bash`. I agree that my solution is a bit hacky, I simply did not find any better way.